packages/web-components/src/divider/divider.stories.ts

```diff
@@ -0,0 +1,65 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { DividerRole, TabsOrientation } from '@microsoft/fast-foundation';
+import { renderComponent } from '../__test__/helpers.js';
+import type { Divider as FluentDivider } from './divider.js';
+import { DividerAlignContent, DividerAppearance } from './divider.options.js';
+import './define.js';
+
+type DividerStoryArgs = Args & FluentDivider;
+type DividerStoryMeta = Meta<DividerStoryArgs>;
+
+const storyTemplate = html<DividerStoryArgs>`
+  <dcs-divider
+    dividerAlignContent=${x => x.dividerAlignContent}
+    dividerAppearance=${x => x.dividerAppearance}
+    dividerRole=${x => x.role}
+    inset=${x => x.inset}
+    orientation=${x => x.orientation}
+  >
+    ${x => x.content}
+  </dcs-divider>
+`;
+
+export default {
+  title: 'Components/Divider',
+  args: {
+    content: 'Text',
+    dividerAlignContent: 'center',
+    dividerAppearance: 'brand',
+    dividerRole: 'presentation',
+    inset: false,
+    orientation: 'horizontal',
+  },
+  argTypes: {
+    dividerAlignContent: {
+      options: Object.keys(DividerAlignContent),
+      control: {
+        type: 'select',
+      },
+    },
+    dividerAppearance: {
+      options: Object.keys(DividerAppearance),
```

Comment:
For these scenarios I think we want to consistently use `Object.values` (though in the case of these the result is the same, consistency will help over time)